tech/languages/r/r-ide.md

```diff
@@ -20,7 +20,7 @@ $ sudo dnf install rstudio-desktop
 
 Then, RStudio will be available in your _Applications_ menu.
 
-![RStudio Desktop](/content/tech/languages/r/rstudio.png)
+[![RStudio Desktop](/content/tech/languages/r/rstudio_downscale.png)](/content/tech/languages/r/rstudio.png)
```

Comment:
That's great!

I didn't think of doing it manualy ... I thought we'd need some 'gallery' plugin. Thanks!